styles.css

```diff
@@ -28,15 +30,8 @@ body {
 
 /* ************************** */
 /* General - primitive - styles */
-h1 {
```

Comment:
We need this. The text over the image on the intro section is now looking too small on desktop and it covers too much of the image on mobile.